contracts/core/fee-calculators/PerformanceFeeCalculator.sol

```diff
@@ -258,6 +262,14 @@ contract PerformanceFeeCalculator is IFeeCalculator {
         } else {
             validateProfitFeePercentage(feePercentage);
 
+            // IMPORATNT: In the case that a profit fee is initially 0 and is set to a non-zero number,
+            // the actualizeFee / updateFeeState function does not update the high watermark
+            // Thus, we need to reset the high water mark here so that users do not pay for profit fees
+            // since inception.
+            uint256 rebalancingSetValue = SetUSDValuation.calculateRebalancingSetValue(msg.sender, oracleWhiteList);
+
+            feeState[msg.sender].lastProfitFeeTimestamp = block.timestamp;
+            feeState[msg.sender].highWatermark = rebalancingSetValue;
```

Comment:
This needs to be checked to make sure its higher than the previous high watermark. We don't want to reduce the high watermark by accident

Reply:
good catch